README.md

```diff
@@ -1,4 +1,9 @@
-# chimera [![Build Status](https://travis-ci.org/personalrobotics/chimera.svg?branch=master)](https://travis-ci.org/personalrobotics/chimera) [![Build Status](https://dev.azure.com/personalrobotics/chimera/_apis/build/status/personalrobotics.chimera?branchName=master)](https://dev.azure.com/personalrobotics/chimera/_build/latest?definitionId=2&branchName=master) [![codecov](https://codecov.io/gh/personalrobotics/chimera/branch/master/graph/badge.svg)](https://codecov.io/gh/personalrobotics/chimera)
+# chimera
+
+[![C/C++ CI](https://github.com/personalrobotics/chimera/workflows/C/C++%20CI/badge.svg)](https://github.com/personalrobotics/chimera/actions)
+[![Build Status](https://travis-ci.org/personalrobotics/chimera.svg?branch=master)](https://travis-ci.org/personalrobotics/chimera)
+[![Build Status](https://dev.azure.com/personalrobotics/chimera/_apis/build/status/personalrobotics.chimera?branchName=master)](https://dev.azure.com/personalrobotics/chimera/_build/latest?definitionId=2&branchName=master)
```

Comment:
Do we still need to test in Azure, or can the above testing be extended to OSX and Windows?

Reply:
The plan is to remove Azure and Travis once Actions covers all the CI build settings they have. We could start removing what Actions covers now I think.